test/src/page_models/access-user-card.js

```diff
@@ -8,6 +8,13 @@ class AccessUserCard {
         this.deleteEdxUserConfirmationDialog = Selector(`#edxUser-${edxUserID} .deleteEdxUserConfirmationDialog`);
         this.cancelUserDeleteButton = Selector(`#edxUser-${edxUserID} .cancelUserDeleteButton`);
         this.confirmUserDeleteButton = Selector(`#edxUser-${edxUserID} .confirmUserDeleteButton`);
+        this.schoolSearchSection = Selector('div').withText('Search a school below to manage their EDX Access');
+        this.selectSchoolDropdown = Selector('#selectInstituteName');
+        this.manageSchoolButton = Selector('#manageSchoolButton');
```

Comment:
The select school dropdown logic should reside in the AccessSchoolUsersPage. Technically it's not in the card.